tests/test_nginx_service.py

```diff
@@ -50,6 +50,29 @@ def test_nginx_type_nodeport(self):
         doc = docs[0]
         assert doc["spec"]["type"] == "NodePort"
 
+    def test_nginx_type_nodeport_specifying_nodeports(self):
```

Comment:
We should also have a check that these fields do not show up with `serviceType: "LoadBalancer"`. We may also need to adjust `nginx/templates/nginx-service.yaml` with logic that only does LB or nodePort, if those are mutually exclusive.